lib/sbi_ipi.c

```diff
@@ -5,6 +5,7 @@
  *
  * Authors:
  *   Anup Patel <anup.patel@wdc.com>
+ *   Nick Kossifidis <mick@ics.forth.gr
```

Comment:
Small typo here. You forgot a ">" here.

Reply:
Oops, missed that one, I just pushed a fix.